Lib/zipimport.py

```diff
@@ -5,7 +5,7 @@
 - ZipImportError: exception raised by zipimporter objects. It's a
   subclass of ImportError, so it can be caught as ImportError, too.
 - _zip_directory_cache: a dict, mapping archive paths to zip directory
```

Comment:
I would argue that this entire point should be removed (and the above `three` reduced to `two`) since this is a private attribute and not included in `__all__`.

Reply:
Agreed, they're private.  Descriptions of the structure of private things belong in comments rather than a module level docstring.